media-gfx/frogr/frogr-1.8.1.ebuild

```diff
@@ -0,0 +1,46 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit gnome.org gnome2-utils meson xdg
+
+DESCRIPTION="flickr applications for GNOME"
+HOMEPAGE="https://live.gnome.org/Frogr"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64"
+
+RDEPEND="
+	>=dev-libs/glib-2.56:2
+	>=dev-libs/json-glib-1.2
+	>=dev-libs/libgcrypt-1.5:=
+	>=dev-libs/libxml2-2.6.8:2
+	media-libs/gstreamer:1.0
+	>=media-libs/libexif-0.6.14
+	>=net-libs/libsoup-2.34:2.4
```

Comment:
Upstream bumped the default dependency for libsoup. It should be:
```
>=net-libs/libsoup-3.0:3.0
```

Reply:
Are you definitely always running `pkgdiff-mg` and reviewing release notes?